### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by obed300**
My first pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# Hello-world4
\ No newline at end of file
+# Hello-world4
+
+> In this project, repository was set to master all of the tools and best practices that I learnt about(Documenting repo in a professional way,Git & GitHub & GitHub Flow and Linters) in previous steps.
+
+
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure the description of the project goes here. you can use [this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Estimated%20time%3A%200.5h-,Description,-In%20this%20project) and change it a little bit

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure you give the instructions on how your repos were set up in the `setup`
- [ ] Kindly give the instruction on how to use this repos once it's set up locally in the `Usage`
- [ ] And delete the checkpoints that did not apply to this project.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @obed300 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856

## Highlights
- [ ] All linters are green

## Required Changes ♻️
- [ ] Kindly consider giving your PR a descriptive title and summary
- [ ] Kindly consider documenting your project in professional way.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jasonfelice — APPROVED** [review]
Hi @obed300, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️
Good documentation✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**